packages/sdk-middleware-http/src/http.js

```diff
@@ -100,17 +100,25 @@ export default function createHttpMiddleware({
   ) => {
     let abortController: any
     const url = host.replace(/\/$/, '') + request.uri
+
+    // if it's true, the client will detect the correct content-type, will be used in sending files (ex: multipart/form-data)
+    const hasNullContentType = request.headers["Content-Type"] === null
     const body =
-      typeof request.body === 'string' || Buffer.isBuffer(request.body)
+      typeof request.body === 'string' ||
+        Buffer.isBuffer(request.body) ||
+        hasNullContentType
```

Comment:
```suggestion
        request.body instanceof FormData
```

Reply:
Thanks @emmenko for feedback, I didn't want to restrict it to FormData as we may need to refactor this again if we introduced a new service with application/x-www-form-urlencoded content type